app/models/setting.rb

```diff
@@ -208,7 +208,7 @@ def self.create!(opts)
   def self.regexp_expand_wildcard_string(string, options = {})
     prefix = options[:prefix] || '\A'
     suffix = options[:suffix] || '\Z'
-    prefix + Regexp.escape(string).gsub('\*', '.*') + suffix
+    prefix + Regexp.escape(string).gsub('\*', '.*').gsub('\?', '[a-zA-Z0-9]') + suffix
```

Comment:
You could unescape the `?`  to allow it:
```suggestion
    prefix + Regexp.escape(string).gsub('\*', '.*').gsub('\?', '?') + suffix
```